src/tss2-esys/api/Esys_HMAC_Start.c

```diff
@@ -25,12 +25,12 @@ static void store_input_parameters (
     esysContext->in.HMAC_Start.handle = handle;
     esysContext->in.HMAC_Start.hashAlg = hashAlg;
     if (auth == NULL) {
-        esysContext->in.HMAC_Start.auth = NULL;
```

Comment:
Commit message: sed -i s/esey/esys

I wish github allowed comments on the commit message itself.